runtime/kusama/src/lib.rs

```diff
@@ -1265,6 +1265,7 @@ impl pallet_xcm::Config for Runtime {
 	type XcmExecuteFilter = OnlyWithdrawTeleportForAccounts;
```

Comment:
do we need to update this?

Reply:
The extrinsic will still work without any update. I decided to leave it alone for now since with the extrinsic it's unnecessary to add a filter.